gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -293,8 +296,18 @@ protected static Path resolvePath(String pattern, String database, String table)
   private Table getTargetTable(Table originTable, Path targetLocation) throws IOException {
     try {
       Table targetTable = originTable.copy();
+      
       targetTable.setDbName(this.targetDatabase);
       targetTable.setDataLocation(targetLocation);
+            
+      // need to update the {@link #HIVE_TABLE_AVRO_SCHEMA_URL} location
+      String oldAvroSchemaURL = targetTable.getTTable().getSd().getSerdeInfo().getParameters().get(HIVE_TABLE_AVRO_SCHEMA_URL);
```

Comment:
This method is intended to copy arbitrary Hive tables, not just Avro tables. These lines are very Avro specific, can they be moved out of this class?